src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/LicenceActivationService.kt

```diff
@@ -88,13 +96,11 @@ class LicenceActivationService(
     return releaseDate <= LocalDate.now()
   }
 
-  private fun LicenceWithPrisoner.isStandardLicenceForActivation(): Boolean {
-    return (
-      licence.licenceStartDate != null &&
-        licence.licenceStartDate!! <= LocalDate.now() &&
-        prisoner.status?.startsWith("INACTIVE") == true
-      )
-  }
+  private fun LicenceWithPrisoner.isStandardLicenceForActivation(): Boolean = (
```

Comment:
linting suggestion

Reply:
As LSD and HDCAD for HDC licences are the same, this logic would work for HDC licences as well rather than interrogating the HDCAD separately. Thanks @SebNegahbanMoJ